docs(support-policy): New Support Policy (#28812)
truecharts/charts: website/src/content/docs/general/support-policy.md

:::tip

Please, always remember to check the content specific to the chart.

:::

We at TrueCharts offer free support via a support-ticketing system on Discord.

## General Support Policy

- We help out where we can, as long as you tried to do it yourself *first*. We do not provide a walkthrough service.
- Our staff will try to help out with providing instructions on how to use `kubectl` and other `*ctl` tools where needed.
- We do not support what we do not build.
- We do not support any Cloud providers or OpenShift.

## ClusterTool Support Policy

- We offer support for all ClusterTool functions, except those under `adv` and `charts`
- If you deviate from default configurations significantly, we cannot guarantee a bootstrap will work and support will be limited on a case-by-case basis
- We do provide support for issues regarding talconfig, as long as ClusterTool is used for the deployment.
- We do provide support for FluxCD setup and layout, as long as ClusterTool is used for the deployment.

## Kubernetes Support Policy

- We do not offer support for any component of kubernets that is not made by TrueCharts.

## Helm Chart Support Policy

> - We offer support for all features available in our helm-charts, including all options in the common-chart.

Reviewer: ```suggestion
- We offer support for all features available in our helm-charts, including all options in the common-chart.
- We offer basic documentation and support for installing, upgrading and editing our Helm-Charts using Helm
```

- We do not support external secret management outside of values.yaml

:::note

Some (common-chart) features might not work correctly with all charts.

:::

## FluxCD Support Policy

> - We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.

Reviewer: ```suggestion
- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.
- We offer basic documentation and support on how to create a `helm-release` file, to deploy our helm-charts on Flux
```

## Structural rules

- If its covered by support as outline above make a ticket in [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826).
- [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) is not rendered on GitHub or any other platform.
- Bugs must be reported on GitHub, but only after confirmation by the staff via a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826)
- Anything not covered by the above Support Policy is not eligble for a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826). If you need assitance outside the scuoe of [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) make a thread in an appropiate [Discord Channel](https://discord.gg/tVsPTHWTtr).

## Expected Prior Knowledge

- How to use a shell/console/terminal and basic commands (ie. `cp`, `ls`, `etc`).
- Basic Kubernetes know how (such as what is a "worker" node vs a "controlplane" node, what's "etcd", what's a "pod" vs a "container", etc).

## What's Explicitly Not Covered By Support:

- Network/Router configuration.
- CloudFlare proxying.

> - Operating-Systems, outside of Talos deployed using ClusterTool.

Reviewer: ```suggestion
- The functioning and compatibility of your self-designed operating systems and kubernetes distros, such as k3s.
```

- Helm 101 (e.g. "how do I install/upgrade/edit a Helm Chart?").
- The software *contained* in our Helm Charts (for example "how to configure Radarr to connect to a torrent client?").

> - Cluster Operations (maintenance, fixing flaky pods, tracking non-deploying fluxcd helm-releases/customisations).

Reviewer: ```suggestion
- Trouble shooting complete clusters, outside of just our Helm-Charts
```

---
title: General Helm Support Policy
sidebar:
  order: 2
---

This manual contains general information on working with TrueCharts Charts using Helm directly.
As well as when editing Helm Yaml for tools like FluxCD.

:::tip

> Please, always remember to check the content specific to the chart.

Reviewer: Can we move the tip to the top?

Otherwise the 2 onelinders are kinda weird

:::

We at TrueCharts offer free support via a support-ticketing system on Discord.

## General Support Policy

> - We help out where we can, as long as you tried to do it yourself first. We do not provide a walkthrough service.

Reviewer: ```suggestion
- We help out where we can, as long as you tried to do it yourself *first*. We do not provide a walkthrough service.
```

> - We expect basic experience with the Linux/POSIX shell (ie. `cp`, `mv`, `ls`).

Reviewer: Duplicate of the below prior knowhow section, can we merge those into this section?
Because its just 3 pointers back there...

> - We do not require you to know `kubectl` and other `*ctl` tools.

Reviewer: ```suggestion
- Our staff will try to help out with providing instructions on how to use `kubectl` and other `*ctl` tools where needed.
```

Maybe something a tiny bit less noob friendly

- We do not support what we do not build.
- We do not support any Cloud providers or OpenShift.

## ClusterTool Support Policy

> - We support all ClusterTool functions, except those under `adv`.

Reviewer: ```suggestion
- We offer support for all ClusterTool functions, except those under `adv` and `charts`
```

Charts is technically just for us/our CI tooling

> - If you deviate from default configurations, we cannot guarantee a bootstrap will work and will not be eligible for support.

Reviewer: ```suggestion
- If you deviate from default configurations significantly, we cannot guarantee a bootstrap will work and support will be limited on a case-by-case basis
```

> - We do provide support for issues regarding talconfig, as long as ClusterTool is used for the deployment.

Reviewer: ```suggestion
- We do provide support for issues regarding talconfig.yaml, as long as ClusterTool is used for the deployment.
- We do provide *basic* assistance with Talos OS, when deployed or maintained through ClusterTool (upgrades, config changes etc)
```

- We do provide support for FluxCD setup and layout, as long as ClusterTool is used for the deployment.

## Kubernetes Support Policy

> - We do not offer support for any component of kubernets that is not made by TrueCharts.

Reviewer: ```suggestion
- We do not offer support for any component of kubernets that is not made by TrueCharts.
- This includes charts by other parties or manually created kubernetes manifests and deployments.
```

## Helm Chart Support Policy

- We offer support for all features available in our helm-charts, including all options in the common-chart.
- We do not support external secret management outside of values.yaml

:::note

> Some (common-chart) features might not work with all charts.

Reviewer: ```suggestion
Some (common-chart) features might not work correctly with all charts.
```

Some (common-chart) features might not work correctly with all charts.

:::

## FluxCD Support Policy

- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.

> ## Structural rules

Reviewer: I think we want to rename this section and move it to the top, after the intro and tip box

- If its covered by support as outline above make a ticket in [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826).
- [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) is not rendered on GitHub or any other platform.
- Bugs must be reported on GitHub, but only after confirmation by the staff via a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826)
- Anything not covered by the above Support Policy is not eligble for a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826). If you need assitance outside the scuoe of [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) make a thread in an appropiate [Discord Channel](https://discord.gg/tVsPTHWTtr).

> ## Expected Prior Knowledge

Reviewer: Maybe move this below the moved-and-renamed structural rules section.
so:
- tip
- intro
- structural rules
- expected prior

> - How to use a shell/console/terminal.
> - Basic shell commands (`cp`, `ls`, `etc`)

Reviewer: ```suggestion
- How to use a shell/console/terminal.
- Basic shell commands (`cp`, `ls`, `etc`)
```

With SCALE this split makes sense, now it doesn't,t can we merge this?

- Basic Kubernetes know how (such as what is a "worker" node vs a "controlplane" node, what's "etcd", what's a "pod" vs a "container", etc).

## What's Explicitly Not Covered By Support:

- Network/Router configuration.
- CloudFlare proxying.
- Operating-Systems, outside of Talos deployed using ClusterTool.

> - Helm 101 (e.g. "how do I install/upgrade/edit a Helm Chart?").

Reviewer: ```suggestion
- Helm 101 (e.g. "how do I install/upgrade/edit a Helm Chart?").
```

I think we can remove this, as we do provide docs for this and provide support if those things dont work.
It basically overlaps the "try yourself first" rule, while also sounding less friendly than we want to be

> - The software *contained* in our Helm Charts (for example "how do I configure Radarr?").

Reviewer: ```suggestion
- The software *contained* in our Helm Charts (for example "how to configure Radarr to connect to a torrent client?").
```

Reviewer: Can we also move this to the Helm-Chart support section?

- Cluster Operations (maintenance, fixing flaky pods, tracking non-deploying fluxcd helm-releases/customisations).

> - How to install `kubectl`/`talosctl`/`flux` CLI.

Reviewer: ```suggestion
- How to install additional tools (like: `kubectl`, `talosctl` etc)
```

:::note

> For all issues not covered by support you can make a thread in an appropriate [Discord Channel](https://discord.gg/tVsPTHWTtr) for assistance from the community.

Reviewer: Can we make this also explicit that for things that are covered, users need to make a support ticket?
Its not a thing to make threads for supported stuff.

Author: Done

> :::note

Reviewer: Maybe move this up with the structural notes?

Author: Wanted this here after I covered what we did not support as another reminder.

Reviewer: Ohh you are correct, its already referenced there.

---

> title: General Helm Support Policy

Reviewer: ```suggestion
title: Support Policy
```

## Expected Prior Knowledge

- How to use a shell/console/terminal and basic commands (ie. `cp`, `ls`, `etc`).
- Basic Kubernetes know how (such as what is a "worker" node vs a "controlplane" node, what's "etcd", what's a "pod" vs a "container", etc).

## What's Explicitly Not Covered By Support

- The functioning and compatibility of your self-designed operating systems and kubernetes distros, such as k3s.
- Troubleshooting complete clusters, outside of just our Helm-Charts.
- CloudFlare proxying.
- Operating-Systems, outside of Talos deployed using ClusterTool.
- The software *contained* in our Helm Charts (for example "how to configure Radarr to connect to a torrent client?").
- Cluster Operations (maintenance, fixing flaky pods, tracking non-deploying fluxcd helm-releases/customisations).
- How to install additional tools (like: `kubectl`, `talosctl` etc)

:::note

If an issue is covered support create a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826) for assistance. For all issues not covered by support you can make a thread in an appropriate [Discord Channel](https://discord.gg/tVsPTHWTtr) for assistance from the community.

:::

> ### Supported Versions of Helm

Reviewer: ```suggestion
## Supported Helm Versions
```

---
title: Support Policy
sidebar:
  order: 2
---

:::tip

Please, always remember to check the content specific to the chart.

:::

TrueCharts is a comprehensive project that focuses on providing opinionated Helm charts for applications to run on Kubernetes-based platforms.

> We also offer free support via a support-ticketing system on Discord.

Reviewer: ```suggestion
We also offer free support via a support-ticketing system on Discord, according to the policy layed out in this document.
```

## Helm Chart Support Policy

- We offer support for all features available in our helm-charts, including all options in the common-chart.
- We offer [basic documentation](../guides/helm/) and support for installing, upgrading and editing our Helm-Charts using Helm.
- We do not support external secret management outside of values.yaml
- We offer support for all our charts trains, except `incubator`, `dev`, and `test`.

:::note

Some [(common-chart)](../common/) features might not work correctly with all charts.

:::

## FluxCD Support Policy

- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.
- We offer [basic documentation](../guides/fluxcd/) and support on how to create a `helm-release` file, to deploy our helm-charts on Flux.
- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.

## What's Explicitly Not Covered By Support

- Troubleshooting complete clusters, outside of just our Helm-Charts.
- CloudFlare proxying.
- The software *contained* in our Helm Charts (for example "how to configure Radarr to connect to a torrent client?").

> - How to install additional tools (like: [kubectl](https://kubernetes.io/docs/tasks/tools/), [talosctl](https://www.talos.dev/v1.8/talos-guides/install/talosctl/), [flux](https://fluxcd.io/flux/cmd/) etc)

Reviewer: ```suggestion
- How to install additional tools (like: [kubectl](https://kubernetes.io/docs/tasks/tools/), [talosctl](https://www.talos.dev/latest/talos-guides/install/talosctl/), [flux](https://fluxcd.io/flux/cmd/) etc)
```

- We offer [basic documentation](../guides/helm/) and support for installing, upgrading and editing our Helm-Charts using Helm.
- We do not support external secret management outside of values.yaml
- We offer support for all our charts trains, except `incubator`, `dev`, and `test`.

:::note

Some [(common-chart)](../common/) features might not work correctly with all charts.

:::

## FluxCD Support Policy

- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.
- We offer [basic documentation](../guides/fluxcd/) and support on how to create a `helm-release` file, to deploy our helm-charts on Flux.
- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.

## What's Explicitly Not Covered By Support

- The functioning and compatibility of your self-designed operating systems and kubernetes distros, such as k3s.
- Troubleshooting complete clusters, outside of just our Helm-Charts.
- CloudFlare proxying.
- The software *contained* in our Helm Charts (for example "how to configure Radarr to connect to a torrent client?").
- Trouble shooting complete clusters, outside of just our Helm-Charts

> - How to install additional tools (like: [`kubectl`](https://kubernetes.io/docs/tasks/tools/), [`talosctl`](https://www.talos.dev/v1.8/talos-guides/install/talosctl/), [`flux`](https://fluxcd.io/flux/cmd/) etc)

Reviewer: ```suggestion
- How to install additional tools (like: [kubectl](https://kubernetes.io/docs/tasks/tools/), [talosctl](https://www.talos.dev/v1.8/talos-guides/install/talosctl/), [flux](https://fluxcd.io/flux/cmd/) etc)
```

I dont think you can have codetags as links...

- We do not support external secret management outside of values.yaml

:::note

Some (common-chart) features might not work correctly with all charts.

:::

## FluxCD Support Policy

- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.
- We offer basic documentation and support on how to create a `helm-release` file, to deploy our helm-charts on Flux.
- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.

## What's Explicitly Not Covered By Support

- The functioning and compatibility of your self-designed operating systems and kubernetes distros, such as k3s.
- Troubleshooting complete clusters, outside of just our Helm-Charts.
- CloudFlare proxying.
- The functioning and compatibility of your self-designed operating systems and kubernetes distros, such as k3s.
- The software *contained* in our Helm Charts (for example "how to configure Radarr to connect to a torrent client?").
- Trouble shooting complete clusters, outside of just our Helm-Charts
- How to install additional tools (like: `kubectl`, `talosctl` etc)

:::note

> If an issue is covered support create a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826) for assistance. For all issues not covered by support you can make a thread in an appropriate [Discord Channel](https://discord.gg/tVsPTHWTtr) for assistance from the community.

Reviewer: ```suggestion
If an issue is covered support create a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826) for assistance. 

For all issues not covered by support you can make a thread in an appropriate [Discord Channel](https://discord.gg/tVsPTHWTtr) for assistance from the community.
```

Missed line break

sidebar:
  order: 2
---

:::tip

Please, always remember to check the content specific to the chart.

:::

TrueCharts is a comprehensive project that focuses on providing opinionated Helm charts for applications to run on Kubernetes-based platforms.

We also offer free support via a support-ticketing system on Discord.

## Support Rules

- If its covered by support as outlined below make a ticket in [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826).
- [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) is not rendered on GitHub or any other platform.
- Bugs must be reported on GitHub, but only after confirmation by the staff via a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826)
- Anything not covered by the below Support Policy is not eligble for a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826). If you need assitance outside the scuoe of [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) make a thread in an appropiate [Discord Channel](https://discord.gg/tVsPTHWTtr).

## Expected Prior Knowledge

> - How to use a shell/console/terminal and basic commands (ie. `cp`, `ls`, `etc`).

Reviewer: ```suggestion
- How to use a shell/console/terminal and basic commands (ie. `cp`, `ls`, `etc`).
- Having read our Guides, News articles and Announcements.
```

---
title: Support Policy
sidebar:
  order: 2
---

:::tip

Please, always remember to check the content specific to the chart.

:::

> This manual contains general information on working with TrueCharts Charts using Helm directly.
> As well as when editing Helm Yaml for tools like FluxCD.
> 
> We at TrueCharts offer free support via a support-ticketing system on Discord.

Reviewer: ```suggestion

TrueCharts is a comprehensive project that focuses on providing opinionated Helm charts for applications to run on Kubernetes-based platforms.

We also offer free support via a support-ticketing system on Discord.
```

## Support Rules

- If its covered by support as outlined below make a ticket in [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826).
- [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) is not rendered on GitHub or any other platform.
- Bugs must be reported on GitHub, but only after confirmation by the staff via a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826)
- Anything not covered by the below Support Policy is not eligble for a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826). If you need assitance outside the scuoe of [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) make a thread in an appropiate [Discord Channel](https://discord.gg/tVsPTHWTtr).

## Expected Prior Knowledge

- How to use a shell/console/terminal and basic commands (ie. `cp`, `ls`, `etc`).

> - Having read our Guides, News articles and Announcements.

Reviewer: ```suggestion
- Having read our Guides, Docs, News and Announcements.
```

## General Support Policy

- We help out where we can, as long as you tried to do it yourself *first*. We do not provide a walkthrough service.
- Our staff will try to help out with providing instructions on how to use `kubectl` and other `*ctl` tools where needed.

> - We do not support what we do not build.

Reviewer: ```suggestion
- Please be aware that the code of conduct also applies to all support
- We do not support what we do not build.
```

Also lets link the new code of conduct page here

## ClusterTool Support Policy

- We offer support for all ClusterTool functions, except those under `adv` and `charts`
- If you deviate from default configurations significantly, we cannot guarantee a bootstrap will work and support will be limited on a case-by-case basis.
- We do provide support for issues regarding talconfig.yaml, as long as ClusterTool is used for the deployment.
- We do provide *basic* assistance with Talos OS, when deployed or maintained through ClusterTool (upgrades, config changes etc)
- We do provide support for FluxCD setup and layout, as long as ClusterTool is used for the deployment.

## Kubernetes Support Policy

- We do not offer support for any component of kubernets that is not made by TrueCharts.
- This includes charts by other parties or manually created kubernetes manifests and deployments.
- We do not offer support for any component of kubernets that is not made by TrueCharts.

## Helm Chart Support Policy

- We offer support for all features available in our helm-charts, including all options in the common-chart.
- We offer basic documentation and support for installing, upgrading and editing our Helm-Charts using Helm.

> - We do not support external secret management outside of values.yaml

Reviewer: ```suggestion
- We do not support external secret management outside of values.yaml
- We offer support for all our charts trains, except `incubator`, `dev, and `test`
```

- We offer support for all our charts trains, except `incubator`, `dev`, and `test`.

:::note

Some (common-chart) features might not work correctly with all charts.

:::

## FluxCD Support Policy

- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.
- We offer [basic documentation](./guides/fluxcd) and support on how to create a `helm-release` file, to deploy our helm-charts on Flux.
- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.

## What's Explicitly Not Covered By Support

- The functioning and compatibility of your self-designed operating systems and kubernetes distros, such as k3s.
- Troubleshooting complete clusters, outside of just our Helm-Charts.
- CloudFlare proxying.
- The software *contained* in our Helm Charts (for example "how to configure Radarr to connect to a torrent client?").

> - Trouble shooting complete clusters, outside of just our Helm-Charts

Reviewer: ```suggestion
```

We also offer free support via a support-ticketing system on Discord.

## Support Rules

- If its covered by support as outlined below make a ticket in [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826).
- [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) is not rendered on GitHub or any other platform.
- Bugs must be reported on GitHub, but only after confirmation by the staff via a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826)
- Anything not covered by the below Support Policy is not eligble for a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826). If you need assitance outside the scuoe of [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) make a thread in an appropiate [Discord Channel](https://discord.gg/tVsPTHWTtr).

## Expected Prior Knowledge

- How to use a shell/console/terminal and basic commands (ie. `cp`, `ls`, `etc`).
- Basic Kubernetes know how (such as what is a "worker" node vs a "controlplane" node, what's "etcd", what's a "pod" vs a "container", etc).

## General Support Policy

> - We help out where we can, as long as you tried to do it yourself *first*. We do not provide a walkthrough service.
> - Our staff will try to help out with providing instructions on how to use `kubectl` and other `*ctl` tools where needed.
> - Please be aware that the code of conduct also applies to all support.

Reviewer: ```suggestion
- We help out where we can, as long as you tried to do it yourself *first*. We do not provide a walkthrough service.
- Our staff will try to help out with providing instructions on how to use `kubectl` and other `*ctl` tools where needed.
- Please be aware that the code of conduct also applies to all support.
```

Maybe move these to the support rules?

> - We do not support what we do not build.

Reviewer: ```suggestion
- We do not support what we do not build.
```

lets get rid of this, I think this new policy is more clear anyway and its not 100% true either

> - We do not support any Cloud providers or OpenShift.

Reviewer: ```suggestion
- We do not support any Cloud providers or OpenShift.
```

move to the excluded stuff?

- We do provide support for issues regarding talconfig.yaml, as long as ClusterTool is used for the deployment.
- We do provide *basic* assistance with [Talos OS](./guides/talos), when deployed or maintained through ClusterTool (upgrades, config changes etc)
- We do provide support for FluxCD setup and layout, as long as ClusterTool is used for the deployment.

## Kubernetes Support Policy

- We do not offer support for any component of kubernets that is not made by TrueCharts.
- This includes charts by other parties or manually created kubernetes manifests and deployments.
- We do not offer support for any component of kubernets that is not made by TrueCharts.

## Helm Chart Support Policy

- We offer support for all features available in our helm-charts, including all options in the common-chart.
- We offer [basic documentation](./guides/helm) and support for installing, upgrading and editing our Helm-Charts using Helm.
- We do not support external secret management outside of values.yaml
- We offer support for all our charts trains, except `incubator`, `dev`, and `test`.

:::note

> Some (common-chart) features might not work correctly with all charts.

Reviewer: ```suggestion
Some (common-chart) features might not work correctly with all charts.
```

Maybe we want to make this a link to the common-chart docs?

:::

## FluxCD Support Policy

- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.
- We offer [basic documentation](./guides/fluxcd) and support on how to create a `helm-release` file, to deploy our helm-charts on Flux.
- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.

## What's Explicitly Not Covered By Support

- The functioning and compatibility of your self-designed operating systems and kubernetes distros, such as k3s.
- Troubleshooting complete clusters, outside of just our Helm-Charts.
- CloudFlare proxying.

> - The functioning and compatibility of your self-designed operating systems and kubernetes distros, such as k3s.

Reviewer: ```suggestion
```

Has become a duplicate

- The software *contained* in our Helm Charts (for example "how to configure Radarr to connect to a torrent client?").
- Trouble shooting complete clusters, outside of just our Helm-Charts

> - How to install additional tools (like: `kubectl`, `talosctl` etc)

Reviewer: ```suggestion
- How to install additional tools (like: `kubectl`, `talosctl` etc)
```

Maybe we can link kubectl and talosctl here explicitly? Might be a nice feature to add while at it anyway.
